src/commands/repo-commands/repo_init.ts

```diff
@@ -10,7 +10,7 @@ const args = {
     type: "string",
   },
   "ignore-branches": {
-    describe: `A list of branches that will never be merged into trunk and that Graphite should ignore when making stacks.`,
+    describe: `A list of branches that Graphite should ignore when constructing stacks (i.e. branches you never intend to merge into trunk).`,
```

Comment:
nit: "tracking" instead of "constructing" since Graphite doesn't actually construct your stacks? Very nitty, but I figure it's more accurate to paint the picture of Graphite as a passive tool for helping with your stacks versus *creating* your stacks (which is more of the type of "magic" framing we've shied away from in the design of the CLI).

```suggestion
    describe: `A list of branches that Graphite should ignore when tracking your stacks (i.e. branches you never intend to merge into trunk).`,
```